api/note.ts

```diff
@@ -65,11 +69,23 @@ export const getNotesOfPubkey =
         return handleResponse<Note[]>(response);
     };
 
-export const getNotes = (urlArg: URL) => async (args: { page: number; limit: number }) => {
+export const getNotes = (urlArg: URL) =>
+async (args: {
+    page: number;
+    limit: number;
+    placeId?: string;
+    sort?: NoteSortByEnum;
```

Comment:
I would prefer to `"recent" | "trending"` instead of Enum because Enum is a runtime value which means more code shipped to the browser.

Not a blocker.